### PR TITLE
fix: can delete item login schema

### PR DESCRIPTION
**Pull request by Defferrard**

### CODE REVIEW

File: src/services/itemLogin/index.ts [hunk 1]
```diff
@@ -167,8 +167,13 @@ const plugin: FastifyPluginAsync = async (fastify) => {
       return db.transaction(async (manager) => {
         const member = asDefined(user?.account);
         assertIsMember(member);
+        const repositories = buildRepositories(manager);
+
+        // Validate permission
+        await itemService.get(user?.account, repositories, itemId, PermissionLevel.Admin);
```
Comment (pyphilia) [inline src/services/itemLogin/index.ts:173]: Use `validatePermission`?

File: src/services/itemLogin/repositories/itemLoginSchema.ts [hunk 2]
```diff
@@ -68,4 +72,21 @@ export class ItemLoginSchemaRepository extends MutableRepository<
 
     return await super.insert({ item: { id: getChildFromPath(itemPath), path: itemPath }, type });
   }
+
+  async deleteOneByItemId(itemId: ItemId) {
+    this.throwsIfParamIsInvalid('itemId', itemId);
+
+    const entity = await this.getOneByItem(itemId);
```
Comment (pyphilia) [inline src/services/itemLogin/repositories/itemLoginSchema.ts:79]:
```suggestion
    const entity = await this.getOneByItemId(itemId);
```

File: src/services/itemLogin/index.ts [hunk 3]
```diff
@@ -167,8 +167,13 @@ const plugin: FastifyPluginAsync = async (fastify) => {
...
+
         try {
-          return (await itemLoginService.delete(member, buildRepositories(manager), itemId)).id;
+          return (await itemLoginService.delete(member, repositories, itemId)).id;
```
Comment (ReidyT) [inline src/services/itemLogin/index.ts:176]:
```suggestion
          const { id } = await itemLoginService.delete(member, repositories, itemId);
          return id;
```

**Review by ReidyT — CHANGES_REQUESTED** [review]
Thanks for this PR 🐍, I agree with what Kim said.

**Pull request closed without merging**